client/state/reducer.js

```diff
@@ -51,6 +52,7 @@ const reducers = {
 	dataRequests,
 	documentHead,
 	experiments,
+	followers,
```

Comment:
How does it sound to make the `followers` reducer modular from the beginning? @sgomes would love that 😉

Reply:
I'm happy to help make it happen, too 🙂 This should help you get started: https://github.com/Automattic/wp-calypso/blob/master/docs/modularized-state.md

Reply:
Yup, I thought about that as well. Will fix that 👍🏻

Reply:
done via 838eb7e